Classes/Parser/ScssParser.php

```diff
@@ -83,10 +84,36 @@ protected function parseFile($file, $settings)
                 'sourceMapBasepath' => '<PATH DOES NOT EXIST BUT SUPRESSES WARNINGS>'
             ]);
         }
-        $css = $scss->compile('@import "' . $file . '"');
 
         $absoluteFilename = GeneralUtility::getFileAbsFileName($file);
-        $relativePath = $settings['cache']['tempDirectoryRelativeToRoot'] . dirname(substr($absoluteFilename, strlen($this->getPathSite()))) . '/';
+
+        // Make paths in url() statements relative to site root
+        $absoluteFilePath = dirname($absoluteFilename);
+        $siteFilePath = dirname(substr($absoluteFilename, strlen($this->getPathSite())));
```

Comment:
There is API to get the web path of an absolute path: `\TYPO3\CMS\Core\Utility\PathUtility::getAbsoluteWebPath`

Reply:
I personally also think that deriving some public asset dir from a (should be private source dir) isn't optimal. At least I'd name the variable to something that represents what is meant, e.g. `$sitePackageThemeWebPath`

Reply:
I don't know why Benji used here the Public path for scss and less, imho this could go into Private and the final css should be public. But that's an other patch...